.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: check-yaml
+    -   id: check-added-large-files
```

Comment:
Do we need more parameters here to make sure our notebooks with images are not having any issues?

Reply:
By default it checks for files >500kB, only for those that have been staged. I guess the docs or manuals files can go over that, but I'm not sure if we're usually checking those in manually, or if they're generated in the CI. If the former, then we can just remove the `check-added-large-files` pre-commit check.

Reply:
I think in that case we should not have this check in this repo. We are building the manual locally and that would be something we should have issue with quiet often? @cqc-alec any other opinion?

Reply:
> I think in that case we should not have this check in this repo. We are building the manual locally and that would be something we should have issue with quiet often? @cqc-alec any other opinion?

Do we have files over 500k in the manual?

Reply:
Just checked, we do have a few. The largest is `docs/manual/.doctrees/environment.pickle` at 2 MB :-(

Reply:
Wonder if it's possible to exclude the `docs` directory.

Reply:
Great!